cmd/wallet-web/src/pages/mixins/common/helper.js

```diff
@@ -89,6 +89,19 @@ export const isVPType = (type) => toLower(type) == 'verifiablepresentation';
 export const getCredentialType = (types) =>
   types.filter((type) => type != 'VerifiableCredential')[0];
 
+// function to replace json path from the json objects
+export function updateCredentialJsonPath(vc, manifestCredential) {
+  let updatedManifestCredential;
+  Object.keys(manifestCredential).forEach(function (key) {
+    updatedManifestCredential = JSON.stringify(
+      manifestCredential,
+      (k, v) => (k == 'path' ? jp.query(vc, v.toString()) : v),
+      1
+    );
+  });
+  return JSON.parse(updatedManifestCredential);
+}
```

Comment:
Can we do something like this?
```js
export function updateCredentialJsonPath(vc, manifestCredential) {
  const updatedManifestCredential = Object.keys(manifestCredential).forEach((key) =>
    JSON.stringify(
      manifestCredential,
      (k, v) => (k == 'path' ? jp.query(vc, v.toString()) : v),
      1
    );
  });
  return JSON.parse(updatedManifestCredential);
}
```

Reply:
const leads to the javascript error for this scenario.

Reply:
What error?